app/controllers/homes_controller.rb

```diff
@@ -1,4 +1,85 @@
 class HomesController < ApplicationController
   def index
+    client = Twitter::REST::Client.new do |config|
+      config.consumer_key        = ENV["TWITTER_CONSUMER_KEY"]
+      config.consumer_secret     = ENV["TWITTER_CONSUMER_SECRET"]
+      config.access_token        = ENV["TWITTER_ACCESS_TOKEN"]
+      config.access_token_secret = ENV["TWITTER_ACCESS_SECRET"]
+    end
+
+    ip_address = request.remote_ip unless Rails.env.test? || Rails.env.development?
+    ip_address = "50.241.127.209" if Rails.env.test? || Rails.env.development?
+    location = GeoIP.new('GeoLiteCity.dat').city(ip_address)
+    @lat = location.latitude
+    @lng = location.longitude
+    q = "geocode:39.5,-98.35,1500mi"
+    all_tweets = client.search(q).take(50)
+    @tweets = all_tweets.delete_if {|t| /(#|)(job|hiring|work)/i.match(t.text)}
+    raw = []
+    @tweets.each do |t|
+      words = t.text.split
+      words.each do |w|
+        raw << w
+      end
+    end
+    @local_trends = raw.find_all { |e| raw.count(e) > 2 && e.length > 3 }.uniq!
+    # binding.pry
+
+    #
+    def consumer_key
+      OAuth::Consumer.new(
+      ENV["TWITTER_CONSUMER_KEY"],
+      ENV["TWITTER_CONSUMER_SECRET"])
+    end
+
+    def access_token
+      OAuth::Token.new(
+      ENV["TWITTER_ACCESS_TOKEN"],
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.